### PR TITLE
feat(tests): Update Fx and Selenium versions on Travis.

### LINKED ISSUE #4270: Travis runs with Fx 40
Current Fx is 49, we can run up to Fx 46 with old selenium w/o updating to geckodriver and marionette.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Use Fx 46 and Selenium 2.53.0

fixes #4270

@jrgm - r?

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
Needs work still. Failing tests on Travis.

**Comment by shane-tomlinson** [comment]
@jrgm - r?

**Comment by jrgm** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -59,5 +59,11 @@ script:
...
   - grunt selectconfig:dist l10n-generate-pages &> /dev/null
   - grunt htmllint:dist
-  - travis_retry npm run test-travis
+  - which firefox
+  - firefox --version
+  # even though Firefox <version> is install in `addons`,
+  # the default firefox-bin is used by Selenium, which is Firefox 31.
+  # firefoxBinary=`which firefox` forces the version we installed
+  # to be used.
+  - travis_retry npm run test-travis firefoxBinary=`which firefox`
```
Comment (vladikoff) [inline .travis.yml]: nice!

**Pull request merged**